### DISCUSSION TITLE
Receiving generic public developer rejections with no useful feedback to correct the issue

### DISCUSSION THREAD

**Discussion by aluminum-ice** [amzn/selling-partner-api-models#2814] [General]
Hi, I have submitted three requests to become a public developer using SP-API. All three have been rejected with a generic message:

> "Hello,
> 
> Thank you for updating your Amazon Selling Partner API (SP-API) Developer Profile.
> 
> We have completed our assessment and have determined that you do not meet criteria for Brand Analytics role access on SP-API. To protect Amazon Customers, we require Developers to have an online business website with publicly available, descriptive information about the analytical products and services provided for Amazon Selling Partners. Services and product must be consistent with section 4.4 and 4.5 of our Acceptable Use Policy.
> 
> No changes have been made to your SP-API access at this time."

We are a start-up wishing to build new products using SP-API. Our products don't yet exist because we need access to the API so we can build them. We've updated our website three times with information about the products we intend to build (our initial website was the typical "stealthy" with minimal information) and each one has resulted in the same generic reply. Without further information, we are simply guessing about what it needed. I've tried re-opening the same case ID with specific questions and received no useful answers. Is there any way to get actual help?

**Comment by AppDev2020-admin** [comment]
We updated our website based on the generic response. But now they say that they will not be giving us the access and their decision is final. They did not even explain reasoning behind their decision.

**Reply by landoncope** [reply]
@AppDev2020-admin - what was your ultimate solution here? I am facing the same issue.

**Comment by AppDev2020-admin** [comment]
Hi Landon,

Sorry I don't have a solution for you. We were also unable to get the role
and had to pivot to publish the app without the role.

**Reply by karanrana-artoon** [reply]
Hi @AppDev2020-admin @landoncope @aluminum-ice 

It's June 2025 now, did you guys get any success for getting Brand Analytics Role access. 
If yes, Please share any additional information (website URL or use-case written) on getting approved.

Thanks in advance 🙏
